prov/sm2/src/sm2_attr.c

```diff
@@ -85,6 +85,19 @@ struct fi_rx_attr sm2_hmem_rx_attr = {
 };
 
 struct fi_ep_attr sm2_ep_attr = {
+	.type = FI_EP_RDM,
+	.protocol = FI_PROTO_SHM,
+	.protocol_version = 1,
+	.max_msg_size = SIZE_MAX,
+	.max_order_raw_size = SIZE_MAX,
+	.max_order_waw_size = SIZE_MAX,
+	.max_order_war_size = SIZE_MAX,
+	.mem_tag_format = FI_TAG_GENERIC,
+	.tx_ctx_cnt = 1,
+	.rx_ctx_cnt = 1,
+};
+
+struct fi_ep_attr sm2_hmem_ep_attr = {
```

Comment:
You will need to undo this change. in your PR... how did you test this?

Reply:
keeping it because we don't have large message device-host